tests/tasks/test_juniperalarmtask.py

```diff
@@ -72,8 +72,8 @@ async def test_task_creates_both_alarm_events_on_both_counts_changed(self, junip
 
         await task.run()
 
-        yellow_event = task.state.events.get(device_name=task.device.name, port="yellow", event_class=AlarmEvent)
-        red_event = task.state.events.get(device_name=task.device.name, port="red", event_class=AlarmEvent)
+        yellow_event = task.state.events.get(device_name=task.device.name, subindex="yellow", event_class=AlarmEvent)
+        red_event = task.state.events.get(device_name=task.device.name, subindex="red", event_class=AlarmEvent)
```

Comment:
Not literal of AlarmType? Might be more trouble than it is worth.

Reply:
Not quite sure what you're getting at. `"red"` and `"yellow"` are valid `AlarmType` values:
```python
AlarmType = Literal["yellow", "red"]
```

Reply:
I was thinking that AlarmType was an enum. Mypy can be a bit stupid about enums, that's why I wondered if t was worth it.